src/main/java/ai/aitia/arrowhead/application/library/util/CoreServiceUri.java

```diff
@@ -11,7 +11,7 @@
  *   AITIA - implementation
  ********************************************************************************/
 
-package eu.arrowhead.client.library.util;
+package ai.aitia.arrowhead.application.library.util;
 
 import org.springframework.util.Assert;
```

Comment:
The core-client-skeleton.jar in the resources is a temporary solution. So before the release we should remove it from the repository.

Reply:
removed